modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -16,11 +16,70 @@ next@docs::server_release_notes.adoc, next@docs_main::server_release_notes.adoc
 
 toc::[]
 
-////
+
 == Changes in 10.15.0
 
 Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.15.0 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
-////
+
+[discrete]
+=== Allow Users to Only Update Own External Storage Configurations
+
+Prevent users from changing external storage configurations not belonging to them. https://github.com/owncloud/core/pull/41225[#41225]
+
+[discrete]
+=== Verify Link in Public Link Notification OCS Share API
+
+The frontend is generating the URL to the public link share and submitting it to the server via the OCS Share API. Via the pure usage of this API any URL could be sent out to any email address (as long as being logged in). This change adds some hardening and validates that the URL is pointing to one of the configured trusted domains. https://github.com/owncloud/core/pull/41214[#41214]
+
+[discrete] 
+=== Sanitize SVG
+
+For the purpose of hardening SVG support, `xlink:href` as well as other `ìmage` tags will be now completely blocked. https://github.com/owncloud/core/pull/41234[#41234]   
+
+[discrete]
+=== Remove IoC Scanner Instructions
+
+IoC scanner instructions, firstly introduced in 10.13.4 and which were displayed during upgrade (console as well as Web Updater) and in the admin settings, have been now completely removed. https://github.com/owncloud/core/pull/41260[#41260]
+
+[discrete]
+=== Fail Fast on Unresponsive Remote Servers
+
+In case a remote/federated server is not responding the share will not be added and any request will now fail faster. Additionally, the checks for reachability now include the port number. https://github.com/owncloud/core/pull/41210[#41210] https://github.com/owncloud/core/pull/41237[#41237]
+
+[discrete]
+=== Handle no Longer Existing Users in Preview Cleanup
+
+A deleted user can still have previews in database (mainly because the final delete job did not complete yet). Such users are no longer being processed so that no exceptions will be generated by the preview cleanup job. https://github.com/owncloud/core/pull/41247[#41247]
```

Comment:
```suggestion
A deleted user can still have previews in the database (mainly because the final delete job did not complete yet). Such users are no longer being processed so that no exceptions will be generated by the preview cleanup job. https://github.com/owncloud/core/pull/41247[#41247]
```